integration_test_suite/integration_tests/src/test/java/io/prometheus/jmx/test/http/ssl/SSLWithTrustStoreAndClientAuth.java

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (C) 2023-present The Prometheus jmx_exporter Authors
```

Comment:
Change the copyright year since this is a new file.

Reply:
done